src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx

```diff
@@ -13,29 +13,57 @@ redirects:
   - /docs/infrastructure/azure-integrations/getting-started/activate-azure-integrations
   - /docs/infrastructure/microsoft-azure-integrations/getting-started/activate-azure-integrations
   - /docs/integrations/microsoft-azure-integrations/getting-started/activate-azure-integrations
-  - /docs/infrastructure/integrations/integrations/azure-integrations
+  - /docs/infrastructure/integrations/integrations/azure-integrations 
 ---
 
-New Relic [infrastructure integrations](/docs/infrastructure/integrations/intro-infrastructure-integrations) allow you to report data from specific systems and supplement [infrastructure's](/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure) default, automatic monitoring. The Microsoft Azure integrations report data from various Azure platform services to your New Relic account. This document explains how to activate Azure integrations.
+For all our solutions for Azure monitoring, see [Introduction to Azure integrations](/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations). 
+
+With our Azure [infrastructure integrations](/docs/infrastructure/integrations/intro-infrastructure-integrations), you can report data to New Relic from dozens of Azure platform services. 
```

Comment:
I'd move this up to the top of the doc. 

Is there a way to make this a little punchier? Can we link to the Azure Marketplace here at the very least?

Reply:
this one isn't related to the Azure marketplace, fyi. might be misudnerstanding...